libr/core/cmd_info.c

```diff
@@ -253,7 +253,7 @@ static int cmd_info(void *data, const char *input) {
 	int is_array = 0;
 	Sdb *db;
 
-	for (i = 0; input[i] && i < 2; i++) {
+	for (i = 0; input[i]; i++) {
```

Comment:
with your patch i think `isrzM*` will not work because * will be beyond the first 2 chars. am i missing somethign here?